lnclient/lnd/lnd.go

```diff
@@ -970,11 +970,12 @@ func (svc *LNDService) GetOnchainBalance(ctx context.Context) (*lnclient.Onchain
 	}, nil
 }
 
-func (svc *LNDService) RedeemOnchainFunds(ctx context.Context, toAddress string, amount uint64, sendAll bool) (txId string, err error) {
+func (svc *LNDService) RedeemOnchainFunds(ctx context.Context, toAddress string, amount uint64, feeRate float64, sendAll bool) (txId string, err error) {
 	resp, err := svc.client.SendCoins(ctx, &lnrpc.SendCoinsRequest{
-		Addr:    toAddress,
-		SendAll: sendAll,
-		Amount:  int64(amount),
+		Addr:        toAddress,
+		SendAll:     sendAll,
+		Amount:      int64(amount),
+		SatPerVbyte: uint64(feeRate),
```

Comment:
Can we use `TargetConf: 1` instead? this might use the highest fee rate? 

```
// ... If 
// neither target_conf, or sat_per_vbyte are set, then the internal wallet will
// consult its fee model to determine a fee for the default confirmation
// target.
```